packages/contentstack-config/src/commands/config/mfa/add.ts

```diff
@@ -0,0 +1,67 @@
+import { cliux, handleAndLogError } from '@contentstack/cli-utilities';
+import { BaseCommand } from '../../../base-command';
+import { MFAService } from '../../../services/mfa/mfa.service';
+import { promptForMFASecret, confirmMFAOverwrite } from '../../../utils/interactive';
+
+export default class AddMFACommand extends BaseCommand<typeof AddMFACommand> {
+  static readonly description = 'Add MFA secret for 2FA authentication';
+
+  static readonly examples = [
+    '$ csdx config:mfa:add',
+  ];
+
+  private readonly mfaService: MFAService;
+
+  constructor(argv: string[], config: any) {
+    super(argv, config);
+    this.mfaService = new MFAService();
+  }
+
+  static readonly flags = {};
+
+  async run(): Promise<void> {
+    try {
+      const envSecret = process.env.CONTENTSTACK_MFA_SECRET;
+      if (envSecret && !this.mfaService.validateSecret(envSecret)) {
+        cliux.error('Invalid secret format in environment variable. The secret must be a valid Base32 string of at least 16 characters.');
+        cliux.print('For more information about MFA, visit: https://www.contentstack.com/docs/developers/security/multi-factor-authentication', { color: 'yellow' });
+        process.exit(1);
+      } else if (envSecret) {
+        cliux.print('Using MFA secret from environment variable');
```

Comment:
@aman19K do we need to inform the user that which .env that we are using here, like path to the env or something, sometime they tend to forget after while, so if they want to update or clean up ,